code/game/objects/items/toys.dm

```diff
@@ -1360,6 +1360,10 @@
 	toysay = "EI NATH!"
 	toysound = 'sound/effects/magic/disintegrate.ogg'
 
+/obj/item/toy/figure/wizard/special
+	toysay = "CLANG!";
+	toysound = 'sound/effects/clang.ogg'
```

Comment:
Making this a subtype means it will spawn in places where we spawn subtypes of action figures like in the box

Reply:
it's a special edition wizard action figure now